addons/assignGear/functions/fnc_assignGearMan.sqf

```diff
@@ -63,21 +63,3 @@ if (isText (_path >> "init")) then {
 _unit setVariable [QGVAR(gearSetup), true, true];
 
 END_COUNTER(assignGearMan);
-
-if (_request) exitWith {}; // don't run hack on requests
-// Stupid hack to fix gear automaticly until we can sort out why setUnitLoadout ocasionly fails on JIPs
-[{
-    params ["_unit", "_loadoutArray"];
-    if (isNull _unit) exitWith {};
-
-    private _arrayUniform = (_loadoutArray select LA_UNIFORM_INDEX) param [0, ""];
-    private _arrayVest = (_loadoutArray select LA_VEST_INDEX) param [0, ""];
-    private _arrayBackpack = (_loadoutArray select LA_BACKPACK_INDEX) param [0, ""];
-    TRACE_6("Checking loadout: Uniform [%1-%2] Vest [%3-%4] Backpack [%5-%6]", _arrayUniform, uniform _unit, _arrayVest, vest _unit, _arrayBackpack, backpack _unit);
-
-    if ((_arrayUniform != (uniform _unit)) || {_arrayVest != (vest _unit)} || {_arrayBackpack != (backpack _unit)}) then {
-        ERROR_2("Mismatch [%1] [%2]", name _unit, typeOf _unit);
-        _unit setUnitLoadout _loadoutArray;
-        ["potato_adminMsg", [(format ["Auto-reset gear on %1", name _unit]), "Server"]] call CBA_fnc_globalEvent;
```

Comment:
I saw this run a few time last session, maybe we should keep it around until we're sure it's working

Reply:
Most of the issues are long loadin (where it runs on the server, but doesn't actually set gear) the one corner case I normally see is DC on loadin so the first init runs, misses the client, and the client post init never runs, leaving a future dude dead at spawn